data/human/transport missions.txt

```diff
@@ -3582,3 +3582,743 @@ mission "Hauler VI cargo prototype 3"
+		conversation
+			`	You guide your ship down to the indicated hanger where there is a motley looking crew waiting to unload your ship. One of them hands you a note. It reads: "Meet me at the Common Grounds cafe in the spaceport if you're interested in doing a project with us. Make sure you're not followed. -Adelita Zero"`
+
+mission "Journalism 2"
+	description "Participate in the guerrilla filming of a Syndicate sweatshop on <destination>."
+	name "Film Sweatshop"
+	source "Moonshake"
+	destination "Maker"
+	passengers 3
+	cargo "stealth drones" 1
+	to offer
+		has "Journalism 1: done"
+
+	on offer
+		conversation
+			`You find the Common Grounds cafe. It has a funky vibe and it's quite busy. You don't seem to be recognized by anyone so you take a seat at a booth and order something to eat. Before your meal arrives a woman, maybe in her late thirties with dark hair, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of, or can even pronounce.`
+			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So, Captain <last> "What did you think of the exposition?"`
+			choice
+				`	"I found it quite moving."`
+				`	"It was OK."`
+					goto wasok
+				`	"It was too political for me."`
+					goto political
+			`	"I'm glad you liked it," she says smiling.`
+				goto inorout
+			label wasok
+			`	"Not exactly a thrilling review, but you're entitled to your opinion," she says smiling.`
+				goto inorout
+			label political
+			`	"Oh. Well unfortunately for many, everything is political. It was political when the Republic tried to block the evacuation. And it was political when the Navy did it without permission. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
+				decline
+			label inorout
+			`	Mariel said you saw me take the photos of the woman and her baby," you are surprised by the bluntness of her admission. "I really like that moment; not many notice it. You're observant but are you kind? We need a ship for our next project but I can't tell you much until you accept. We need you to transport a team to one planet, quickly land on another, then pickup the team and return here with everyone safe and sound. Nothing to dicey. But it's a dangerous universe, so who knows. Oh, and we have to leave today. It will pay, of course. 200000 credits. So what do you say? Are you in or out?" Your orders are served and you eat in silence and consider her proposition while she drinks her smoothie.`
```

Comment:
```suggestion
			`	"Mariel said you saw me take the photos of the woman and her baby." You're surprised by the bluntness of her admission. "I really like that moment; not many notice it. You're definitely observant. We need a ship for our next project, but I can't tell you much until you accept. We need you to transport a team to one planet, quickly land on another, then pickup the team and return here with everyone safe and sound. Nothing too dicey. But it's a dangerous universe, so who knows? Oh, and we have to leave today. It will pay, of course. 200,000 credits. So, what do you say? Are you in or out?" Your orders are served, and you eat in silence and consider her proposition while she drinks her smoothie.`
```
A meal is a long time to go without answering. You might want to change up the last sentence.